src/dash-table/utils/actions.js

```diff
@@ -1,12 +1,20 @@
 import * as R from 'ramda';
+import merge from 'ramda/es/merge';
```

Comment:
Use `R.merge` instead

Reply:
`R.merge` is deprecated - `mergeRight` is its more explicit replacement.

Reply:
Interesting, that makes for **a lot** of `R.merge` usage to be updated in the table repo later :)